src/ng/compile.js

```diff
@@ -1509,6 +1511,17 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
                 });
             }
 
+            // support (event), (^event), on-event, and onbubble-event bindings
```

Comment:
Maybe a workaround for this is to ignore attributes which are present in the bindings object